### PR TITLE
Updates to SDK metrics ref, add Temporal Client info

### PR DESCRIPTION
**Pull request by flossypurse**
Preview: https://deploy-preview-966--mystifying-fermi-1bc096.netlify.app/docs/reference/sdk-metrics

### CODE REVIEW

File: docs/reference/metrics.md [hunk 1]
```diff
@@ -1,83 +1,395 @@
...
+
+A local activity execution failed
+
+- Type: Counter
+- Go, Java
+- Available keys: `activity_type`, `namespace`, `task_queue`
+
+### `local_activity_execution_latency`
+
+The time it is taking for Local Activity Executions to complete, from the time the first Activity Task is generated, to the time the SDK responds that the execution is complete.
```
Comment (djmagee) [inline docs/reference/metrics.md:159]:
```suggestion
Time to complete a Local Activity Execution, from the time the first Activity Task is generated to the time the SDK responds that the execution is complete.
```

File: docs/reference/metrics.md [hunk 2]
```diff
@@ -1,83 +1,395 @@
...
+
+Temporal Client made an RPC long poll request.
+
+- Type: Counter
+- Core-Client, Go, Java
+- Available keys: `namespace`, `operation`
+
+### `long_request_latency`
+
+Temporal Client gRPC long poll request latency.
```
Comment (djmagee) [inline docs/reference/metrics.md:199]:
```suggestion
Latency of a Temporal Client gRPC long poll request.
```

File: docs/reference/metrics.md [hunk 3]
```diff
@@ -1,83 +1,395 @@
...
+
+Temporal Client failed an RPC request.
+
+- Type: Counter
+- Available in: Core, Go, Java
+- `namespace`, `operation`
+
+### `request_latency`
+
+Temporal Client gRPC request latency.
```
Comment (djmagee) [inline docs/reference/metrics.md:239]:
```suggestion
Latency of a Temporal Client gRPC request.
```

File: docs/reference/metrics.md [hunk 4]
```diff
@@ -1,83 +1,395 @@
...
+
+A Workflow Task did not find a cached Workflow Worker.
+
+- Type: Counter
+- Available in: Core, Go, Java
+- Available keys: `namespace`, `task_queue`
+
+### `sticky_cache_size`
+
+Current cache size in number of Workflow Executions.
```
Comment (djmagee) [inline docs/reference/metrics.md:263]:
```suggestion
Current cache size, expressed in number of Workflow Executions.
```

File: docs/reference/metrics.md [hunk 5]
```diff
@@ -1,83 +1,395 @@
...
+
+A Workflow Execution has been forced from the cache intentionally.
+
+- Type: Counter
+- Available in: Go, Java
+- Available keys: `namespace`, `task_queue`
+
+### `unregistered_activity_invocation`
+
+There was a request to spawn an Activity Execution that is not registered with the Worker.
```
Comment (djmagee) [inline docs/reference/metrics.md:279]:
```suggestion
A request to spawn an Activity Execution is not registered with the Worker.
```

File: docs/reference/metrics.md [hunk 6]
```diff
@@ -1,83 +1,395 @@
...
+
+- Type: Counter
+- Available in: Go
+- Available keys: `activity_type`, `namespace`, `task_queue`, `workflow_type`
+
+### `worker_start`
+
+A Worker Entity has been registered/created/started.
```
Comment (djmagee) [inline docs/reference/metrics.md:287]:
```suggestion
A Worker Entity has been registered, created, or started.
```

File: docs/reference/metrics.md [hunk 7]
```diff
@@ -1,83 +1,395 @@
...
+
+- Type: Counter
+- Available in: Core, Go, Java
+- Available keys: `namespace`, `task_queue`, `worker_type`
+
+### `worker_task_slots_available`
+
+The total number of Workflow Task and Activity Task execution slots that are currently available.
+The `worker_type` key provides the ability to differentiate respectively (Workflow Workers execute Workflow Tasks, Activity Workers execute Activity Tasks).
```
Comment (djmagee) [inline docs/reference/metrics.md:296]:
```suggestion
Use the `worker_type` key to differentiate execution slots.
(Workflow Workers execute Workflow Tasks; Activity Workers execute Activity Tasks.)
```

File: docs/reference/metrics.md [hunk 8]
```diff
@@ -1,83 +1,395 @@
...
+
+- Type: Gauge
+- Available in: Go, Java
+- Available keys: `namespace`, `task_queue`, `worker_type`
+
+### `workflow_active_thread_count`
+
+Total amount of Workflow threads in the Worker Process
```
Comment (djmagee) [inline docs/reference/metrics.md:304]:
```suggestion
Total amount of Workflow threads in the Worker Process.
```

File: docs/reference/metrics.md [hunk 9]
```diff
@@ -1,83 +1,395 @@
...
+
+A Workflow Execution completed successfully.
+
+- Type: Counter
+- Available in: Core, Go, Java
+- Available keys: `namespace`, `task_queue`, `workflow_type`
+
+### `workflow_continue_as_new`
+
+A workflow ended continued-as-new
```
Comment (djmagee) [inline docs/reference/metrics.md:327]:
```suggestion
A Workflow ended with Continue-As-New.
```

File: docs/reference/metrics.md [hunk 10]
```diff
@@ -1,83 +1,395 @@
...
+
+- Type: Counter
+- Available in: Core, Go, Java
+- Available keys: `namespace`, `task_queue`, `workflow_type`
+
+### `workflow_endtoend_latency`
+
+Workflow total execution time in milliseconds
```
Comment (djmagee) [inline docs/reference/metrics.md:335]:
```suggestion
Workflow total execution time in milliseconds,
```

File: docs/reference/metrics.md [hunk 11]
```diff
@@ -1,83 +1,395 @@
...
+
+- Type: Histogram
+- Available in: Core, Go, Java
+- Available keys: `namespace`, `task_queue`, `workflow_type`
+
+### `workflow_failed`
+
+A workflow ended failed
```
Comment (djmagee) [inline docs/reference/metrics.md:343]:
```suggestion
A Workflow failed.
```

File: docs/reference/metrics.md [hunk 12]
```diff
@@ -1,83 +1,395 @@
...
+
+- Type: Counter
+- Available in: Core, Go, Java
+- `namespace`, `task_queue`, `workflow_type`
+
+### `workflow_task_execution_failed`
+
+A workflow task execution failed
```
Comment (djmagee) [inline docs/reference/metrics.md:351]:
```suggestion
A Workflow Task Execution failed.
```

File: docs/reference/metrics.md [hunk 13]
```diff
@@ -1,83 +1,395 @@
...
+
+- Type: Counter
+- Available in: Core, Go, Java
+- `namespace`, `task_queue`, `workflow_type`
+
+### `workflow_task_execution_latency`
+
+Workflow task execution time in milliseconds
```
Comment (djmagee) [inline docs/reference/metrics.md:359]:
```suggestion
Workflow Task Execution time in milliseconds.
```

File: docs/reference/metrics.md [hunk 14]
```diff
@@ -1,83 +1,395 @@
...
+
+- Type: Histogram
+- Available in: Core, Go, Java
+- Available keys: `namespace`, `task_queue`, `workflow_type`
+
+### `workflow_task_queue_poll_empty`
+
+A Workflow Worker polled a Task queue and timed out without picking up a Workflow Task.
```
Comment (djmagee) [inline docs/reference/metrics.md:367]:
```suggestion
A Workflow Worker polled a Task Queue and timed out without picking up a Workflow Task.
```

File: docs/reference/metrics.md [hunk 15]
```diff
@@ -1,83 +1,395 @@
...
+
+A Workflow Worker polled a Task Queue and successfully picked up a Workflow Task.
+
+- Type: Counter
+- Available in: Core, Go, Java
+- Available keys: `namespace`, `task_queue`, `workflow_type`
+
+### `workflow_task_replay_latency`
+
+Time it takes to catch up on replaying a Workflow Task in milliseconds.
```
Comment (djmagee) [inline docs/reference/metrics.md:383]:
```suggestion
Time to catch up on replaying a Workflow Task in milliseconds.
```

**Review by djmagee — COMMENTED** [review]
Lotsa little tweaks. Verify whether my additions to `activity_execution_latency` match intent.

File: docs/reference/metrics.md [hunk 16]
```diff
@@ -1,83 +1,393 @@
...
+
+Number of Signals whose payload could not be deserialized.
+
+- Type: Counter
+- Available in: Go, PHP, Java
+- Available keys: `namespace`, `task_queue`, `workflow_type`
+
+### `local_activity_execution_cancelled`
+
+A local activity execution was canceled.
```
Comment (aarohib) [inline docs/reference/metrics.md:140]:
```suggestion
A local Activity Execution was canceled.
```

File: docs/reference/metrics.md [hunk 17]
```diff
@@ -1,83 +1,393 @@
...
+
+- Type: Counter
+- Available in: Go, PHP, Java
+- Available keys: `activity_type`, `namespace`, `task_queue`
+
+### `local_activity_execution_failed`
+
+A Local Activity Execution failed.
```
Comment (aarohib) [inline docs/reference/metrics.md:148]:
```suggestion
A local Activity Execution failed.
```

**Review by aarohib — APPROVED** [review]

**Pull request merged**